app/experimenter/experiments/migrations/0164_auto_20210401_1758.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 3.1.7 on 2021-04-01 17:58
+
+from django.db import migrations, models
+
+from experimenter.experiments.constants import NimbusConstants
```

Comment:
Oh yeah don't import modules into migrations.  Models you look up using `get_model` and constants you just hard code as literals like so

```py
from django.db import migrations, models


def update_changelogs(apps, schema_editor):
    NimbusChangeLog = apps.get_model("experiments", "NimbusChangeLog")
    db_alias = schema_editor.connection.alias
    NimbusChangeLog.objects.using(db_alias).filter(old_status="Review").update(
        old_status="Draft",
        old_publish_status="Review",
    )
    NimbusChangeLog.objects.using(db_alias).filter(old_status="Accepted").update(
        old_status="Draft",
        old_publish_status="Waiting",
    )
    NimbusChangeLog.objects.using(db_alias).filter(new_status="Review").update(
        new_status="Draft",
        new_publish_status="Waiting",
    )
    NimbusChangeLog.objects.using(db_alias).filter(new_status="Accepted").update(
        new_status="Draft",
        new_publish_status="Waiting",
    )
```